llvm/tools/llvm-lsp/README.md

```diff
@@ -0,0 +1,27 @@
+# LLVM LSP server
+
+## Build
+
+```bash
+cmake -S llvm -B buildR -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++
+ninja -j 6 -C buildR llvm-lsp-server
```

Comment:
Here and elsewhere:

Can we use `cmake --build` to be consistent with other documentation? E.g. https://llvm.org/docs/CMake.html#quick-start